custom-ingestion/Chart.py

```diff
@@ -0,0 +1,22 @@
+from Snapshot import Snapshot
+
+
+class Chart(Snapshot):
+    snapshot_type = 'Chart'
```

Comment:
the `__class__ `property will yield the same result our of the box, no need to define this variable :P